Sources/reactant/InitCommand.swift

```diff
@@ -67,76 +104,56 @@ class InitCommand: Command {
-            mainSources.append(Source(path: "Sources/Components/Main/MainRootView.swift", type: .source(mainRootViewXml)))
-            mainSources.append(Source(path: "Sources/Components/Main/MainRootView.ui.xml", type: .file(mainRootViewXmlUI)))
-        } else {
-            mainSources.append(Source(path: "Sources/Components/Main/MainRootView.swift", type: .source(mainRootView)))
+        if readBool(title: "Enable Torch ORM?") {
+            experimentalFeatures.insert(.torch)
         }
 
-        mainSources.append(contentsOf: [
-            Source(path: "Sources/Components/Main/MainController.swift", type: .source(mainController)),
-            Source(path: "Sources/Models", type: .directory),
-            Source(path: "Sources/Services", type: .directory),
-            Source(path: "Sources/Styles/General.styles.xml", type: .file(generalStyles)),
-            Source(path: "Sources/Wireframes/MainWireframe.swift", type: .source(mainWireframe)),
-            Source(path: "Sources/AppDelegate.swift", type: .source(appDelegate(experimentalFeatures: experimentalFeatures))),
-            Source(path: "Sources/DependencyModule.swift", type: .source(dependencyModule)),
-            Source(path: "Sources/ApplicationModule.swift", type: .source(applicationModule)),
-            Source(path: "Resources/Assets.xcassets/AppIcon.appiconset/Contents.json", type: .file(assetsJson), ignoreXcode: true),
-            Source(path: "Resources/Assets.xcassets/Contents.json", type: .file(assetsJson), ignoreXcode: true),
-            Source(path: "Resources/Assets.xcassets", type: .resourceRef),
-            Source(path: "Resources/LaunchScreen.storyboard", type: .resource(launchScreen)),
-            Source(path: "Resources/Localizable.strings", type: .resource(gitKeep)),
-            Source(path: "Resources/Info.plist", type: .fileRef),
-        ])
-
-        var targets = [
-            ResolvedTarget(
-                name: productName,
-                sources: Sources(items: mainSources, root: applicationDir),
-                type: .main)
-        ]
-        if false && readBool(title: "Enable Unit Tests?") {
-            let unitTestDir = workingDir.appending(outputDir).appending(components: "UnitTests")
-            targets.append(
-                ResolvedTarget(
-                    name: "\(productName)UnitTests",
-                    sources: Sources(items: [], root: unitTestDir),
-                    type: .unitTest)
-            )
-        }
-        if false && readBool(title: "Enable UI Tests?") {
-            let uiTestDir = workingDir.appending(outputDir).appending(components: "UITests")
-            targets.append(
-                ResolvedTarget(
-                    name: "\(productName)UITests",
-                    sources: Sources(items: [], root: uiTestDir),
-                    type: .uiTest)
-            )
+        if false && readBool(title: "Enable Cuckoo - boilerplate free mocking framework?") {
```

Comment:
Doesn't this effectively comment out the code inside the `if`? Why is it done so?